src/utils/index.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+/**
+ * HTML element attributes.
+ */
+export
+interface IElementOptions {
+  /**
+   * A space-delimited list of classes for the element.
+   */
+   className?: string;
+
+   /**
+    * The title attribute of the element.
+    */
+   title?: string;
+
+   /**
+    * The value attribute of the element.
+    */
+   value?: string;
+}
+
+/**
+ * Create an HTML element with given attributes and children.
+ * 
+ * #### Notes
+ * As a convenience, the children can be a list of elements
+ * or a string (which is interpreted as the textContent of the
+ * new element).
+ */
+
+export
+function createElement(tag: string, attr?: IElementOptions, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
I think it would be *less* magical to do something like this:

`createElement(tag: string, className: string, content?: HTMLElement[] | string)`

Trying to handle value and title is just going to clutter this up and make it unreadable, I'd say leave it out.